widget/embedded/src/constants/languages.ts

```diff
@@ -31,7 +51,12 @@ export const LANGUAGES: LanguageItem[] = [
   { title: 'French', label: 'Français', local: 'fr', SVGFlag: French },
   { title: 'Japanese', label: '日本語', local: 'ja', SVGFlag: Japanese },
   { title: 'Portuguese', label: 'Português', local: 'pt', SVGFlag: Portuguese },
-  { title: 'Chinese', label: '中国人', local: 'zh', SVGFlag: Chinese },
```

Comment:
Please make sure the related languages are displayed near each other. For example

- lang 1
- lang 2
- chinese (1)
- chinese (2)
- lang 3
-

Reply:
I sorted the languages in alphabetical order.